### PR TITLE
Fix:  improve logging

### PR DESCRIPTION
**Pull request by w7shdev**
# Issue Addressed #77 

The issue was some of the logs wasn't appear if the session fails 
with this PR we tackle this issue and closing #77 

## Description:

**Before** 

Screen Shot 2022-01-22 at 8 49 22 PM

**After**

Screen Shot 2022-01-22 at 8 50 43 PM

## Checklist

- [ ] I have commented the newly added code
- [ ] I update the Readme.md file to include the new changes to the project
- [ ] All the previously included Unit Tests passed in all the cases

### CODE REVIEW

File: src/WC_Gateway_ThawaniGateway.php [hunk 1]
```diff
@@ -444,12 +444,14 @@ public function process_payment($order_id)
...
         $this->logger('📝Session creation Failed');
-        $this->logger('📝Response Code: ' . $response->code);
-        $this->logger('📝Response Description: ' . $response->description);
+        $this->logger('📝Response log');
+        foreach($response as $key => $value) { 
+            $this->logger("📝Response {$key} : "  . $value);
+        }
         $this->logger('📝Success URL: ' . $this->api->get_redirect_uri($response->data->session_id));
-        $this->logger('📝Cancel URL: ' . $this->get_return_url($order));
+        $this->logger('📝Cancel URL: ' . $this->get_return_url($order));
```
Comment (w7shdev) [inline src/WC_Gateway_ThawaniGateway.php:454]:
```suggestion
        $this->logger('📝Cancel URL: ' . $this->get_return_url($order));
```

File: src/WC_Gateway_ThawaniGateway.php [hunk 2]
```diff
@@ -444,10 +444,12 @@ public function process_payment($order_id)
                 'redirect' => $this->api->get_redirect_uri($response->data->session_id),
             );
         }
-
+
```
Comment (w7shdev) [inline src/WC_Gateway_ThawaniGateway.php:447]:
```suggestion

```

**Comment by w7shdev** [comment]
@HajerSubhi @Ma7eer  👀

**Comment by HajerSubhi** [comment]
@w7shdev I'll review it in the afternoon.

**Review by HajerSubhi — APPROVED** [review]
LGTM

**Pull request merged**